dashboard/app/models/pd/enrollment.rb

```diff
@@ -47,14 +47,19 @@ class Pd::Enrollment < ActiveRecord::Base
   validates_confirmation_of :email
   validates_email_format_of :email, allow_blank: true
 
-  validate :validate_school_name, unless: :skip_school_validation
-  validates_presence_of :school_info, unless: :skip_school_validation
```

Comment:
Do there exist rows with this attribute set? If so, there will be a data cleanup to remove those references?